x-pack/plugins/maps/public/classes/sources/es_search_source/update_source_editor.js

```diff
@@ -42,6 +44,9 @@ export class UpdateSourceEditor extends Component {
     termFields: null,
     sortFields: null,
     supportsClustering: false,
+    supportsMvt: false,
+    mvtDisabledReason: null,
+    clusteringDisabledReasin: null,
```

Comment:
Why was `clusteringDisabledReasin` added to state? Its never read any where

Reply:
typo: https://github.com/elastic/kibana/blob/828ce8a19b37c6aceced94635196931144928dee/x-pack/plugins/maps/public/classes/sources/es_search_source/update_source_editor.js#L106